readme.md

```diff
@@ -43,14 +48,55 @@ npm install remark-language-server
 
 Usage of this package depends on your editor integration.
 Because this is based on
-[`vscode-languageserver-node`][vscode-languageserver-node], the same transports
+[`unified-languageserver-node`][unified-languageserver-node], the same features
 are supported.
 
-Practical examples are coming soon.
-
 `remark-language-server` uses the same configuration files as
 [`remark-cli`][remark-cli].
 
+## Examples
+
+### Emacs
+
+Use [`lsp-mode`][lsp-mode] to use the remark language server with Emacs.
+
+### Neovim
+
+The remark language server is part of [`nvim-lspconfig`][nvim-lspconfig], a
+collection of common configurations for Neovim’s built-in language server
+client.
+This means you can add remark language server in one line:
+
+```lua
+require'lspconfig'.remark_ls.setup{}
+```
+
+Detailed installation instructions can be found [here](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#remark_ls).
```

Comment:
`here` is not very accessible (screenreaders allow jumping between links and then they read the label, so “here”). Also, maybe use a reference/definition?

Reply:
```suggestion
`nvim-lspconfig` has [detailed installation instructions](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#remark_ls) such as configuration options.
```